tests/random_fuzzer.cpp

```diff
@@ -572,15 +652,18 @@ bool fuzz_running(size_t N) {
     for (size_t k = 0; k < size; k++) {
       input[k] = char(distribution(generator));
     }
-    if (!fuzz_this(input.data(), size)) {
-      dump_case();
+    if(!run_test(input.data(), size)) {
       return false;
     }
   }
   printf("\n");
   return true;
 }
 
+bool precomputed() {
+  return run_test("\x06\xd8\x00\x00\x0a\x00\x3f\x00", 8);
+}
+
```

Comment:
I rewrote this to:
```cpp
  const unsigned char cleaned_crash[]
      = {0xff, 0x20, 0x20, 0x20, 0x20, 0xd8, 0x00, 0x00, 0x20, 0x20, 0x20, 0x20};
  const unsigned int cleaned_crash_len = 12;

  return run_test((const char *) cleaned_crash, cleaned_crash_len);

```
and then I got:

> runtime error: subtraction of unsigned offset from 0x51700000008c overflowed to 0x5170000000a7

in src/icelake/icelake_convert_utf16_to_utf8.inl.cpp:183

seems like "inlen" is 0, then adjust becomes -31 converted to a size_t.

Reply:
I just took minimized data from a fuzzer crash, so not the same data as you had.